frontend/src/components/question-creation/QuestionElementItem.tsx

```diff
@@ -39,12 +39,20 @@ const renderQuestionContent = (content: QuestionElement) => {
           key={id}
           data={data as MultipleChoiceData}
           id={id}
+          isMultiSelect={false}
+        />
+      );
+    case QuestionElementType.MULTI_SELECT:
+      return (
+        <MultipleChoiceElement
+          key={id}
+          data={data as MultipleChoiceData}
```

Comment:
would you be able to rename the type to something more general since we are using it for both cases? maybe something like `MultiOptionData`!